Examples/basic.md

```diff
@@ -0,0 +1,59 @@
+This setup allows sending notifications/alerts, with TLS setups and spoof protection. This can also be a beginner's guide to get a feeling about how the setup works.
+
+First, get your TLS certificate from [Letsencrypt](https://letsencrypt.org) or follow a docker example [here](https://www.humankode.com/ssl/how-to-set-up-free-ssl-certificates-from-lets-encrypt-using-docker-and-nginx). Do not forget to also add your subdomain `mail.mydomain.org` to the list of domains during creating the certificate.
+
+Then, use the following `docker-compose.yml` file (edit with your own domain name) instead of the default one to re-create your mailserver following the instructions on the main page (or simply re-run `sudo docker-compose up -d mail`).
+
+```
+version: '2'
```

Comment:
putting the docker-compose.yml into the readme makes it much harder to maintain it. as you need to copy it out and your changes back in. Rather put the yml alongside of the readme.